doc/manual/configuration.rst

```diff
@@ -1015,13 +1015,18 @@ Plugins are loaded in the same order as listed here. For each name in this
 section there must be a .py-file in the ``plugins`` directory next to the
 recipes. For a detailed description of plugins see :ref:`extending-plugins`.
 
-User configuration (default.yaml)
----------------------------------
+User configuration (default.yaml / .bob-settings.yaml)
```

Comment:
I'd like to go even one step further an do it like git and be xdg-compatible: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html. Quoting git:

```
FILES
       If not set explicitly with --file, there are four files where git config will search for configuration options:

       $(prefix)/etc/gitconfig
           System-wide configuration file.

       $XDG_CONFIG_HOME/git/config
           Second user-specific configuration file. If $XDG_CONFIG_HOME is not set or empty, $HOME/.config/git/config will be used. Any
           single-valued variable set in this file will be overwritten by whatever is in ~/.gitconfig. It is a good idea not to create this file
           if you sometimes use older versions of Git, as support for this file was added fairly recently.

       ~/.gitconfig
           User-specific configuration file. Also called "global" configuration file.

       $GIT_DIR/config
           Repository specific configuration file.
```
For Bob this would be:
* `/etc/bobdefault.yaml`
* `$XDG_CONFIG_HOME/bob/default.yaml` resp. `~/.config/bob/default.yaml`
* `./default.yaml`